_posts/2019-11-23-devfest-rj-2019.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: 'DevFest Rio de Janeiro 2019'
+date_from: 2019-11-23
+date_to: 2019-11-23
+description: 'O DevFest é um evento de grande porte criado por e para desenvolvedores e interessados em tecnologia.'
+link: 'https://www.sympla.com.br/devfest-rio-de-janeiro__680060'
+tags:
+- gdg
```

Comment:
Pode deixar apenas a tag "geral" mesmo :)

Reply:
Esse evento veio junto quando fiz o merge das alterações do repositório original com o meu repositório local. No entanto, posso fazer a alteração se for o caso